tests/components/frontend/test_init.py

```diff
@@ -336,3 +337,24 @@ async def test_auth_authorize(mock_http_client):
     resp = await mock_http_client.get(authorizejs.groups(0)[0])
     assert resp.status == 200
     assert "public" in resp.headers.get("cache-control")
+
+
+async def test_get_version(hass, hass_ws_client):
+    """Test get_version command."""
+    frontend = await async_get_integration(hass, "frontend")
+    cur_version = next(
+        req.split("==", 1)[1]
+        for req in frontend.requirements
+        if req.startswith("home-assistant-frontend==")
+    )
+
+    await async_setup_component(hass, "frontend", {})
+    client = await hass_ws_client(hass)
+
+    await client.send_json({"id": 5, "type": "frontend/get_version"})
+    msg = await client.receive_json()
+
+    assert msg["id"] == 5
+    assert msg["type"] == TYPE_RESULT
+    assert msg["success"]
+    assert msg["result"] == {"version": cur_version}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5418277)